src/eterna/ChatManager.ts

```diff
@@ -62,8 +63,48 @@ export default class ChatManager {
         } else {
             this._chatbox.classList.add('hidden');
         }
+
+        this.chatShowSignal.emit({show, bound: this.getChatboxBounds(show)});
     }
 
+    private getChatboxBounds(show: boolean): {
```

Comment:
Would using getBoundingClientRect allow this calculation to be simplified?

Reply:
Yup. I fixed.